functional_tests/test_simple_list_creation.py

```diff
@@ -29,9 +30,10 @@ def test_can_start_a_list_and_retrieve_it_later(self):
         # and now the page lists "1: Buy peacock feathers" as an item in a
         # to-do list table
         inputbox.send_keys(Keys.ENTER)
+        time.sleep(1)
```

Comment:
probably not needed

Reply:
but without this sleep this test failed locally because the url has not changed yet